src/Illuminate/Http/FileHelpers.php

```diff
@@ -46,6 +48,6 @@ public function hashName($path = null)
             $path = rtrim($path, '/').'/';
         }
 
-        return $path.md5_file($this->getRealPath()).'.'.$this->guessExtension();
+        return $path.Uuid::uuid4()->toString().'.'.$this->guessExtension();
```

Comment:
ramsey/uuid just uses random_bytes() too, see [`RandomBytesGenerator`](https://github.com/ramsey/uuid/blob/master/src/Generator/RandomBytesGenerator.php).

In fact my code above is just the removal of a bunch of classes, factories, generators, etc. the result is the equivalent code without all the winter layers.